lang/python/python-schedule/Makefile

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (C) 2019 CZ.NIC, z. s. p. o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-schedule
+PKG_VERSION:=0.6.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=schedule-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/s/schedule/
+PKG_HASH:=f9fb5181283de4db6e701d476dd01b6a3dd81c38462a54991ddbb9d26db857c9
+
```

Comment:
I'd remove whitespace here.

Reply:
I see that there is no whitespace?

Reply:
@neheb kindly reminder. This is the last blocker.

Reply:
I mean make it like so:

PKG_SOURCE:=schedule-$(PKG_VERSION).tar.gz
PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/s/schedule/
PKG_HASH:=f9fb5181283de4db6e701d476dd01b6a3dd81c38462a54991ddbb9d26db857c9
PKG_BUILD_DIR:=$(BUILD_DIR)/schedule-$(PKG_VERSION)

Reply:
Ok, that's not a problem at all. As you wish.